cmd/proxy-server/server/server.go

```diff
@@ -231,6 +234,43 @@ func WithDebugPort(addr string) Option {
 	})
 }
 
+func WithPrometheusPort(addr string) Option {
```

Comment:
How about we name this WithMetricsPort? The format originally came from Prometheus but it's used by more things now.

Please add a function comment too.

Reply:
Done